lib/matplotlib/animation.py

```diff
@@ -1013,6 +1013,12 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
            on to the `savefig` command which is called repeatedly to
            save the individual frames.
 
+        progress_callback : function, optional
+            Is a function with two parameters allowing to monitor the
```

Comment:
A callback function that will be called for every frame to notify the
progress of the saviing. It must have the signature ::

    def func(current_frame: int, total_frames: int) -> Any

where *current_frame* is the current frame number and *total_frames*
is the total number of frames to be saved. Return values may exist but
are ignored.

    progress_callback=lambda i, n: print(f'Saving frame {i} of {n}')